app/src/androidTest/java/org/simple/clinic/patient/PatientRepositoryAndroidTest.kt

```diff
@@ -3171,4 +3172,83 @@ class PatientRepositoryAndroidTest {
     val savedPrescribedDrugs = database.prescriptionDao().getAllPrescribedDrugs()
     assertThat(savedPrescribedDrugs).containsExactlyElementsIn(prescribedDrugs)
   }
+
+  @Test
+  fun when_a_patient_is_called_then_it_should_be_retrieving_contact_patient_profile_correctly() {
```

Comment:
Can we rename this test to `loading_contact_patient_profile_should_work_correctly` or `fetching_the_contact_patient_profile_should_load_the_patient_details` instead to be consistent with other tests, please?

Reply:
Not necessary in this PR, you can do it in the next PR.